setup.py

```diff
@@ -15,7 +17,7 @@
     version=about['__version__'],
     license=about['__license__'],
     description=about['__description__'],
-    long_description=open('README.md').read(),
+    long_description=open('README.md', encoding="utf-8").read(),
```

Comment:
This will break on Python 2 with `TypeError: 'encoding' is an invalid keyword argument for this function`